test_browserstack_module_boundary fails on main: Playwright install step lacks -pl -am

## Summary

`tests.scripts.test_browserstack_module_boundary.BrowserStackModuleBoundaryTest.test_workflows_resolve_sdk_only_for_browserstack_jobs` fails on `origin/main`.

The test asserts every `mvn ` line in `.github/workflows/e2eTests.yml` contains `-pl <module> -am`. The Playwright browser-install step does not:

```
run: mvn -f shaft-engine/pom.xml -e org.codehaus.mojo:exec-maven-plugin:3.6.3:java "-Dexec.mainClass=com.microsoft.playwright.CLI" "-Dexec.args=install --with-deps chrome"
```

```
AssertionError: '-pl shaft-engine -am' not found in 'run: mvn -f shaft-engine/pom.xml -e org.codehaus.mojo:exec-maven-plugin:3.6.3:java ...'
```

## Evidence this is pre-existing

Both files the test reads are byte-identical to `origin/main`:

```bash
git diff --stat origin/main HEAD -- .github/workflows/e2eTests.yml tests/scripts/test_browserstack_module_boundary.py
# (no output)
```

Found while running the full `tests/scripts` suite for an unrelated agent-guidance change (469 tests, this the only failure). Not fixed there to avoid scope creep.

## Likely resolution

Either the assertion should skip `-f <module>/pom.xml` invocations (which already scope the reactor and do not need `-pl -am`), or the install step should adopt the `-pl ... -am` form. The first looks correct: `-f` and `-pl` are alternative scoping mechanisms.

--- a/tests/scripts/test_browserstack_module_boundary.py
+++ b/tests/scripts/test_browserstack_module_boundary.py
@@ -1,4 +1,5 @@
 import json
+import re
 import unittest
 import xml.etree.ElementTree as ET
 from pathlib import Path
@@ -68,7 +69,9 @@ class BrowserStackModuleBoundaryTest(unittest.TestCase):
 
         self.assertTrue(commands)
         for command in commands:
-            if "exec:java" in command:
+            if "exec:java" in command or re.search(
+                r"exec-maven-plugin:[^ ]+:java(?:\s|$)", command
+            ):
                 self.assertIn("-f shaft-engine/pom.xml", command)
                 self.assertNotIn("shaft-browserstack", command)
                 continue

